code/modules/projectiles/gun_attachables.dm

```diff
@@ -1756,3 +1767,36 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	for(var/X in master_gun.actions)
 		var/datum/action/A = X
 		A.update_button_icon()
+
+
+/obj/item/attachable/tp44_longbarrel
```

Comment:
dont use numbers in typepaths

Reply:
Wait why itd the gun name

Reply:
As I said in disc0rd its for consistency and co

Reply:
dunno what consistency, alot of other typepaths use number, like ak47, m16, g22, etc